online-data-tools/build_www_manifest.py

```
@@ -0,0 +1,74 @@
+#!/usr/bin/env -S uv run --no-project --script
+# /// script
+# requires-python = ">=3.10"
+# ///
+"""Emit `www/manifest.json` advertising the day-rotated SQLite databases.
+
+The www/ directory holds 1–2 `<YYYY-MM-DD>.db` files. This script discovers
```

Ruff (0.15.17)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)